src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -325,7 +325,7 @@ else if (moduleName.startsWith("com.")) {
             fullyQualifiedClassName = moduleName;
         }
         else {
-            final String path = SLASH_PATTERN.matcher(filePath).replaceAll("\\.");
+            final String path = SLASH_PATTERN.matcher(filePath).replaceAll(".");
```

Comment:
This didn't need to be escaped.

Reply:
it is confirmed https://docs.oracle.com/javase/8/docs/api/java/util/regex/Matcher.html#replaceAll-java.lang.String-